content/en/docs/tasks/administer-cluster/kms-provider.md

```diff
@@ -31,6 +41,17 @@ as the Kubernetes control plane, is responsible for all communication with the r
 To configure a KMS provider on the API server, include a provider of type `kms` in the
 `providers` array in the encryption configuration file and set the following properties:
 
+### KMS v1
+* `name`: Display name of the KMS plugin.
+* `endpoint`: Listen address of the gRPC server (KMS plugin). The endpoint is a UNIX domain socket.
+* `cachesize`: Number of data encryption keys (DEKs) to be cached in the clear.
+  When cached, DEKs can be used without another call to the KMS;
+  whereas DEKs that are not cached require a call to the KMS to unwrap.
+* `timeout`: How long should `kube-apiserver` wait for kms-plugin to respond before
+  returning an error (default is 3 seconds).
+
+### KMS v2
+* `apiVersion`: API Version for KMS provider (Allowed values: v2, v1 or empty. Any other value will result in an error.)
```

Comment:
Not sure if this is correct.

Reply:
https://github.com/kubernetes/website/pull/35046#discussion_r942699082